glide/src/main/java/org/odk/collect/glide/ImageLoaderImpl.kt

```diff
@@ -12,25 +12,24 @@ import com.bumptech.glide.request.target.Target
 import org.odk.collect.glide.svg.SvgSoftwareLayerSetter
 import java.io.File
 
-object ImageLoader {
+class ImageLoaderImpl : ImageLoader {
```

Comment:
This could be named `GlideImageLoader` rather than needing the `Impl` suffix. I think we should probably also rename the module to `imageloader` (or something similar) as the beauty of moving this all to a module and hiding the implementation behind an interface is that dependent modules don't even need to know that Glide exists.

Reply:
Done.